zerver/lib/external_accounts.py

```diff
@@ -23,13 +24,13 @@
     "twitter": {
         "text": "Twitter",
         "url_pattern": "https://twitter.com/%(username)s",
-        "name": "Twitter username",
+        "name": str(gettext_lazy("Twitter username")),
```

Comment:
You need to remove the `str()` calls -- they make the string no longer lazy.

Reply:
I am getting this error if I pass without `str()`
```
TypeError: Object of type '__proxy__' is not JSON serializable
```

Reply:
Will it work if I assign `gettext_lazy("Twitter username")` to some variable first?